<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSelectMultipleAccounts</class>
 <widget class="QWidget" name="FormSelectMultipleAccounts">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>151</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>151</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>151</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_academicAdmin">
      <property name="text">
       <string>Academic Administrator</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_instructor">
      <property name="text">
       <string>Instructor</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_TATM">
      <property name="text">
       <string>Teacher/Marker Assistant</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_adminAssist">
      <property name="text">
       <string>Administrator Assistant</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="pushButton_sysAdmin">
      <property name="text">
       <string>System Administrator</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="AccountType_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Account Type</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
